build.gradle

```diff
@@ -13,6 +13,7 @@ buildscript {
     dependencies {
         classpath "com.github.jengelman.gradle.plugins:shadow:${shadowJarVersion}"
         classpath "com.jfrog.bintray.gradle:gradle-bintray-plugin:${bintrayVersion}"
+        classpath 'com.netflix.nebula:nebula-publishing-plugin:4.9.1'
```

Comment:
and remove the              
`url "https://oss.sonatype.org/content/repositories/snapshots/"` in the repositories section. I can't comment on that line,,,